### ISSUE TITLE
Richards Request 26-04

### ISSUE THREAD

**Issue by adamshelley31** [Pro-Enviro/IDNI#70]
1. ~~On the fuel data can you please call it something like ‘Non Half-Hourly data, (ie other non-grid electricity or gas costs)’ Just to clarify what HH is~~
2. ~~On the upload page can you add a text box to explain how they go about this (in laymans terms), where they find it and how they do it~~
3. On the Level2 PET please add the same info and option to upload here as  well.  
4. ~~On the Dashboard the box asking to  select company and add data isn’t explained or clear what this means is for, again could we add a bit of explanation here as well~~
5. ~~The energy questions are back on the sign up page.  Remember you had simplified this.~~
6. ~~The company commute is still at the top level on the level 2 PET .  Could you move this to this next level down under the company travel costs?  It is confusing at the minute because it is asking a cost but no costs are associated.~~
7. ~~On the innovation, training and export %age questions in the PET it just needs the column to let them input their %.  The other column doesn’t really mean anything to them and the % is the useful thing for us.~~

**Comment by adamshelley31** [comment]
@mila-proenviro - Can you test no. 6 when you have a free moment?

**Comment by mila-proenviro** [comment]
Issue 6, was tested on branch "richards-requests-26-04" and it is working.
